vars/mailer.groovy

```diff
@@ -6,34 +6,48 @@ def sendEmailFailure() {
             recipientProviders: [[$class: 'DevelopersRecipientProvider'], [$class: 'RequesterRecipientProvider']])
 }
 
-def sendEmail_failedPR() {
+def sendEmail_failedPR(pr_type) {
     emailext(
-            subject: 'Pull request #$ghprbPullId of $ghprbGhRepository: $ghprbPullTitle failed',
-            body: 'Pull request #$ghprbPullId of $ghprbGhRepository: $ghprbPullTitle  FAILED\n' +
-                  'Build log: ${BUILD_URL}consoleText\n' +
-                  'Failed tests ${TEST_COUNTS,var="fail"}: ${BUILD_URL}testReport\n' +
-                  '(IMPORTANT: For visiting the links you need to have access to Red Hat VPN. In case you don\'t have access to RedHat VPN please download and decompress attached file.)',
+            subject: "${pr_type} #$ghprbPullId of $ghprbGhRepository: $ghprbPullTitle failed",
+            body:  "Pull request #$ghprbPullId of $ghprbGhRepository: $ghprbPullTitle  FAILED \n" +
+                    "Build log: ${BUILD_URL}consoleText \n" +
+                    "Failed tests \${TEST_COUNTS,var=\"fail\"}: ${BUILD_URL}testReport \n" +
+                    "(IMPORTANT: For visiting the links you need to have access to Red Hat VPN. In case you don\'t have access to RedHat VPN please download and decompress attached file.)",
             attachmentsPattern: "error.log.gz",
-            to: emailextrecipients ([[$class: 'DevelopersRecipientProvider'], [$class: 'RequesterRecipientProvider']]))
+            recipientProviders: [[$class: 'DevelopersRecipientProvider'], [$class: 'RequesterRecipientProvider']])
+}
+
+def sendEmail_unstablePR(pr_type) {
+    emailext(
+            subject: "${pr_type} #$ghprbPullId of $ghprbGhRepository: $ghprbPullTitle was unstable",
+            body:  "Pull request #$ghprbPullId of $ghprbGhRepository: $ghprbPullTitle was UNSTABLE \n" +
+                    "Build log: ${BUILD_URL}consoleText \n" +
+                    "Failed tests \${TEST_COUNTS,var=\"fail\"}: ${BUILD_URL}testReport \n" +
+                    "(IMPORTANT: For visiting the links you need to have access to Red Hat VPN) \n" +
+                    "*********************************************************************************************************************************************************** \n" +
+                    "${FAILED_TESTS}",
+            recipientProviders: [[$class: 'DevelopersRecipientProvider'], [$class: 'RequesterRecipientProvider']])
 }
 
-def sendEmail_unstablePR() {
+def sendEmail_fixedPR(pr_type) {
```

Comment:
the same here https://github.com/kiegroup/kogito-runtimes/blob/f4c84914b945e56c34d20f3ea88c725d7bfab968/Jenkinsfile#L87